test/api/libs/responseVerifyer.py

```diff
@@ -221,15 +221,47 @@ def verify_teleporter_zip(self, teleporter_archive: bytes):
 		return self.errors
 
 
+	# Check if a string is a valid IPv4 address
+	def valid_ipv4(self, addr: str) -> bool:
+		octets = addr.split(".") # type: list[str]
+		if len(octets) != 4:
+			return False
+		for octet in octets:
+			if not octet.isdigit():
+				return False
+			if int(octet) < 0 or int(octet) > 255:
+				return False
+		return True
+
+
+	# Check if a string is a valid IPv6 address
+	def valid_ipv6(self, addr: str) -> bool:
+		# Split the address into parts
+		parts = addr.split(":") # type: list[str]
+		# Check if the address is a valid IPv6 address
+		if len(parts) != 8:
```

Comment:
Is it possible to use shortened IPv6 addresses here like `2001:db8::1234:5678`? 
Will this code work in this case?

Reply:
Thanks for your comment, something got lost here. As you can see, I added `import ipaddress` above and actually have also used it for the check but this got somehow lost. I will add it back in